Basic/Invaders/Assets/Scripts/InvadersGame.cs

```diff
@@ -147,6 +147,8 @@ public override void OnNetworkDespawn()
             m_Enemies.Clear();
             m_Shields.Clear();
         }
+
+        NetworkManager.OnClientConnectedCallback -= OnClientConnected;
```

Comment:
Just a comment to use NetworkManager.Singleton here and below for parity in the codebase.